PREPARATION.md

```diff
@@ -22,10 +22,20 @@ For general deployment, the SCIM bridge requires these components:
 
 You'll need to create a DNS record with your SCIM bridge domain name after the bridge is deployed. Follow the steps in each respective deployment guide to finish setting up your DNS record at the appropriate time.
 
+> **Note**
+>
+>Container-as-a-Service platforms (for example, Azure Container Apps, DigitalOcean App Platform) include TLS endpoint management, so you do not need to create a DNS record for these deployments.
```

Comment:
I think we can remove the hyphenation for CaaS and the capitalization here and in other instances. Also, it should be plural without "platforms". Usage elsewhere varies, but [Atlassian](https://www.atlassian.com/microservices/cloud-computing/containers-as-a-service) doesn't use hyphens and neither does [IBM](https://www.ibm.com/topics/containers-as-a-service) so I'd rather we stick to that. Right now it reads a little strange. I've also added "and" instead of a comma, and a contraction instead of "do not".

```suggestion
>Containers as a service, such as Azure Container Apps and DigitalOcean App Platform, include TLS endpoint management, so you don't need to create a DNS record for these deployments.
```

Reply:
I'm in agreement with the superfluous hyphens, but "Container as a Service platforms" reads much better and is more appropriate to me than "Containers as a Service". Azure Container Apps or DigitalOcean App Platform are each platforms that allow customers to host Containers as a Service.

Reply:
I'm happy to use that instead :)

Reply:
Combining both of your suggestions here. 

```suggestion
>Containers as a service platforms, such as Azure Container Apps and DigitalOcean App Platform, include TLS endpoint management, so you don't need to create a DNS record for these deployments.
```